content/docs/rfcs/32/README.md

```diff
@@ -518,85 +531,108 @@ Also for RLN we do a single secret component input for the circuit.
 Thus we need to hash the secret array (two components) to a secret hash,
 and we use that as a secret component input.
 
-# Apendix C: Auxiliary tooling
+# Appendix C: Auxiliary tooling
 
 There are few additional tools implemented for easier integrations and usage of the RLN protocol.
 
+[`zerokit`](https://github.com/vacp2p/zerokit) is a set of Zero Knowledge modules, written in Rust and designed to be used in many different environments.
+Among different modules, it supports `Semaphore` and `RLN`.
+
 [`zk-kit`](https://github.com/appliedzkp/zk-kit) is a typescript library which exposes APIs for identity credentials generation,
 as well as proof generation.
-It supports various protocols (`Semaphore`, `RLN`),.
+It supports various protocols (`Semaphore`, `RLN`).
 
 [`zk-keeper`](https://github.com/akinovak/zk-keeper) is a browser plugin which allows for safe credential storing and proof generation.
 You can think of MetaMask for ZK-Proofs.
 It uses `zk-kit` under the hood.
 
-# Apendix D: Example usage
+# Appendix D: Example usage
+
+The following examples are code snippets using the `zerokit` RLN module.
```

Comment:
I think we probably want to keep the PSE library here? It should be updated though since it no longer lives in zk-kit. Can check with PSE team if they want to update this in this PR or separately or how we should proceed to make spec draft

Maybe we can have two appendices?